eventrecorder/recorder_integration_test.go

```diff
@@ -0,0 +1,105 @@
+//go:build integration
+
+package eventrecorder_test
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"net/http"
+	"os"
+	"testing"
+	"time"
+
+	"github.com/filecoin-project/lassie-event-recorder/eventrecorder"
+	"github.com/filecoin-project/lassie/pkg/types"
+	"github.com/jackc/pgx/v5/pgtype"
+	"github.com/jackc/pgx/v5/pgxpool"
+	"github.com/stretchr/testify/require"
+)
+
+func TestPostEvent(t *testing.T) {
+	var dbdsn = "postgres://postgres:postgres@localhost:5432/SaturnLassieEvents"
```

Comment:
this test expects the database to already exist?

Reply:
yep created via postgres container [here](https://github.com/filecoin-project/lassie-event-recorder/pull/18/files#diff-c15aca62594cd4128d2975cc11183817cdafd4db05edd3138382c3e450b96ab1R14)